### DISCUSSION TITLE
NetBox Version Command

### DISCUSSION THREAD

**Discussion by kopacko** [netbox-community/netbox#9545] [General]
Is there a command that would output the version of NetBox, similar to like what `postgres -V` does?

**Comment by cpund** [comment]
Depends how you're wanting to do it. One way would be to use `curl http://127.0.0.1:8000/api/status/` or using Python via bash `python3 -c "import requests; print(requests.get('http://localhost:8000/api/status/').json()['netbox-version'])"`. The former gives you an entire readout of status without parsing out netbox-version, but the latter will do exactly that.

Or, the third, if you have jq installed, ` curl -s http://127.0.0.1:8000/api/status/ | jq '."netbox-version"'`

**Comment by kopacko** [comment]
Yea I am trying to do it via CLI and I tried all the ways you suggested but I get this:

`{'detail': 'Authentication credentials were not provided.'}`

**Comment by markkuleinio** [comment]
`grep ^VERSION /opt/netbox/netbox/netbox/settings.py`

**Comment by kopacko** [comment]
@markkuleinio You are a beautiful person! Thank you!

**Comment by jbakklund** [comment]
Starting from NetBox 4.1; to identify installed version of NetBox, while NetBox is not running, you can use
`grep version /opt/netbox/netbox/release.yaml | grep -o '[0-9.]*`
